apps/meteor/app/channel-settings/server/methods/saveRoomSettings.ts

```diff
@@ -333,12 +355,13 @@ Meteor.methods({
 				delete settings.retentionIgnoreThreads;
 			}
 		});
+		keys = Object.keys(settings);
 
 		// saving data
-		for await (const setting of Object.keys(settings)) {
+		keys.forEach((setting) => {
 			const value = settings[setting];
 
-			const saver = await settingSavers[setting];
+			const saver = settingSavers[setting];
 			if (saver) {
 				saver({
```

Comment:
One of the savers is async... But it was already being called without await so it's not a blocker for this PR - but probably a bug.